### ISSUE TITLE
poem-openapi cannot find attribute `has_permissions` in this scope

### ISSUE THREAD

**Issue by jsamtsang** [DDtKey/poem-grants#4]

**Comment by jsamtsang** [comment]
Error caused by annotating the document

![image](https://github.com/DDtKey/poem-grants/assets/31495837/9020343f-e2f1-4325-896a-6cccc3ef0860)

After deleting it, it's OK.

But this is a bug！！！

**Comment by DDtKey** [comment]
Thanks for reporting @jsamtsang!

**Comment by DDtKey** [comment]
Hi @jsamtsang, it was fixed by #5.

As a fast workaround to add doc-comment, you can place it under the attribute:
```rs
#[has_permissions("user:page")]
#[oai(...)]
/// My doc comment
async fn page(...) -> R { .. }
```

Or wait for `1.0.0-beta.3` version to be released.

**Comment by DDtKey** [comment]
Fix was released, thanks again @jsamtsang !
